lib/aruba/environments/unix_environment.rb

```diff
@@ -0,0 +1,11 @@
+require 'aruba/environments/basic_environment'
+
+module Aruba
+  module Environments
+    class UnixEnvironment < BasicEnvironment
+      def match?(*)
+        Aruba::Platform.unix?
```

Comment:
Mmh... The order of the `<<` is important. `find` walks the collection "top down" and stops when the block returns true. I use this pattern in a few projects and it works. Not sure why this fails here. 

```ruby
objs = []
objs << 1
objs << 2

objs.find { |o| o > 0 }

objs = []
objs << 2
objs << 1

objs.find { |o| o > 0 }
```
Running this in pry:

```
[7] pry(main)> objs = []
=> []
[8] pry(main)> objs << 1
=> [1]
[9] pry(main)> objs << 2
=> [1, 2]
[10] pry(main)>
[11] pry(main)> objs.find { |o| o > 0 }
=> 1
[12] pry(main)>
[13] pry(main)> objs = []
=> []
[14] pry(main)> objs << 2
=> [2]
[15] pry(main)> objs << 1
=> [2, 1]
[16] pry(main)>
[17] pry(main)> objs.find { |o| o > 0 }
=> 2
```